Source/WebCore/accessibility/AccessibilityRenderObject.cpp

```diff
@@ -1498,6 +1497,9 @@ AXTextRuns AccessibilityRenderObject::textRuns()
         }
         currentLineIndex = newLineIndex;
 
+        // Within each iteration of this loop, we are looking at the *next* text box to compare to the current.
+        // So, we need to set the textRunDomOffsets after the line index comparison, in order to assign the right DOM offsets per Text Box.
```

Comment:
"Text Box" is capitalized here, but not in the previous sentence. The uncapitalized version is probably right?